pkg/csi_driver/gcfs_driver.go

```diff
@@ -133,13 +169,18 @@ func NewGCFSDriver(config *GCFSDriverConfig) (*GCFSDriver, error) {
 			csi.ControllerServiceCapability_RPC_CREATE_DELETE_SNAPSHOT,
 		}
 		driver.addControllerServiceCapabilities(csc)
+
+		if config.FeatureOptions.FeatureStateful != nil && config.FeatureOptions.FeatureStateful.Enabled {
+			runMultishareReconciler(config)
```

Comment:
This looks like an antipattern in this file.

All object init goes inside NewGCFSDriver(), and all factory start resides in gcfs_driver Run() function